lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -115,10 +115,13 @@ end |> write_beam
 
 defmodule ExUnit.DocTestTest.NoImport do
   @doc """
-  iex> ExUnit.DocTestTest.NoImport.min(1, 2)
+  iex> ExUnit.DocTestTest.NoImport.max(1, 2)
+  {:ok, 2}
+
+  iex> max(1, 2)
   2
   """
-  def min(a, b), do: max(a, b)
```

Comment:
well, the added value of this PR is that it is testing a function with a colliding name that is not being imported. I haven't found a test tests this, but it may deserve a separate test, which means we will have to change the lines anyway.